Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM docker.io/debian:buster-slim
+
+
+# Install base system dependencies
+RUN apt update
+# https://stackoverflow.com/questions/58160597/docker-fails-with-sub-process-usr-bin-dpkg-returned-an-error-code-1
+RUN mkdir -p /usr/share/man/man1 
+RUN apt install -y --no-install-recommends uwsgi uwsgi-plugin-python3 python3 python3-setuptools pipenv gcc gettext \
+    libldap2-dev libsasl2-dev libgpgme-dev python3-dev libgraphviz-dev graphviz libmagic-dev libjs-mathjax \
+    libpcre3 libpcre3-dev make yui-compressor nginx supervisor
```

Comment:
Some of these dependencies should be removed after the build to reduce the image size. I suppose the following can be removed:
* python3-setuptools
* pipenv
* gcc
* make
* yui-compressor
* everything that ends in -dev (even though for some of them, the non-dev package probably has to be installed, e.g. gpg)

This will probably reduce the size by several hundred MB.

Reply:
Also, libpcre3 is not needed at all.

Reply:
python3-setuptools is needed by sentry but other than that I agree